### PR TITLE
fix: correct remaining time

### PR DESCRIPTION
**Pull request by vansangpfiev**
## Describe Your Changes
Remaining time of library sometimes does not work. Implement our own function.

image

## Fixes Issues

- https://github.com/janhq/cortex.cpp/issues/1655

### CODE REVIEW

**Comment by namchuai** [comment]
could you please add some unit test

**Comment by vansangpfiev** [comment]
> could you please add some unit test

Updated.

File: engine/utils/format_utils.h [hunk 1]
```diff
@@ -112,4 +112,40 @@ inline std::string BytesToHumanReadable(uint64_t bytes) {
   out << std::fixed << std::setprecision(2) << result << " " << unit;
   return out.str();
 }
+
+inline std::string TimeDownloadFormat(int seconds) {
+  // Constants for time units
+  const uint64_t kSecondsInMinute = 60;
```
Comment (namchuai) [inline engine/utils/format_utils.h]: nit: should using `constexpr`?

**Review by namchuai — APPROVED** [review]
✅ lgtm

**Pull request merged**